app/views/devise/registrations/new.html.erb

```diff
@@ -12,12 +12,17 @@
 <section class="pb-5" id="registration_new">
   <div class='container'>
     <div class="row">
-      <div class="col-md-6 mx-auto card p-5">
+      <div class="col-md-6 mx-auto card p-5">        
         <%= bootstrap_form_for(resource, as: resource_name,
                      url: registration_path(resource_name)) do |f| %>
 
           <%= render "devise/shared/error_messages", resource: resource %>
 
+          <%# adopter_account built in registrations controller #new %>
+          <%= f.fields_for :adopter_account do |fields| %>
+            <%= fields.hidden_field :user_id %>
+          <% end %>
+
```

Comment:
`user_id` appears to always be blank for me? I think we can do this on the backend too 
```
signup_params
 ...
 .merge(adopter_account_attributes: { user_id: "" })
```

Reply:
I we remove the code below from the form (which is what was missing) then rails will not know to create the associated record on that request.
```
<%# adopter_account built in registrations controller #new %>
          <%= f.fields_for :adopter_account do |fields| %>
            <%= fields.hidden_field :user_id %>
          <% end %>
```